scripts/aws_ecs_benchmarks_cdn.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# make sure the following line is added to `~/.bashrc`
+source "$HOME/.cargo/env"
+
+# assign local ip
+ip=`curl http://169.254.169.254/latest/meta-data/local-ipv4`
```

Comment:
Might be worth a comment that these IPs can change if the infrastructure in AWS changes.  So we shouldn't rely on these always being accurate.

Reply:
I think this one is actually the AWS metadata server: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

Worth a comment about it though

Reply:
@rob-maron This is nice

Reply:
Need to Enable Task Metadata Endpoint by setting the ECS_ENABLE_CONTAINER_METADATA to true in AWS console. Therefore we plan to just do `AWS_METADATA_IP=ip` in the scripts for better readability.